src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -40,39 +40,83 @@
 public final class ReadsSparkSource implements Serializable {
     private static final long serialVersionUID = 1L;
     private static final String HADOOP_PART_PREFIX = "part-";
-    public static final int DEFAULT_SPLIT_SIZE = 10485760;
+    public static final int DEFAULT_SPLIT_SIZE = 10 * 1024 * 1024; // 10MB
 
     private transient final JavaSparkContext ctx;
     public ReadsSparkSource(JavaSparkContext ctx) {
         this.ctx = ctx;
     }
 
+    private static long getInputSizeInBytes(final JavaSparkContext ctx, final String filePath) {
```

Comment:
I don't think this method should be buried inside `ReadSparkSource` like this -- could you move it to a public method in a utility class (`SparkUtils`?), and add a direct unit test for it?